src/hooks/useWelcomeBanner.ts

```diff
@@ -18,38 +19,47 @@ interface UseWelcomeBannerReturn {
 export const useWelcomeBanner = ({
   account,
   isFunded,
+  isLoadingBalances,
 }: UseWelcomeBannerProps): UseWelcomeBannerReturn => {
   const welcomeBannerBottomSheetModalRef = useRef<BottomSheetModal>(null);
-  const [bannerPresented, setBannerPresented] = useState(false);
+  const [bannerPresented, setBannerPresented] = useState(true);
```

Comment:
it looks like this `bannerPresented` state is not being used anymore anywhere. If so, could we remove it to clean the code?